test/common/memory_usage.h

```diff
@@ -86,7 +90,9 @@ namespace utils {
     /* Returns 0 if not implemented on platform. */
     std::size_t GetMemoryUsage(MemoryStatType stat = currentUsage) {
         utils::suppress_unused_warning(stat);
-#if __TBB_WIN8UI_SUPPORT || defined(WINAPI_FAMILY)
+#if __TBB_TEST_USE_ADDRESS_SANITIZER
+        return __sanitizer_get_current_allocated_bytes();
```

Comment:
Can you please add a comment why we need to use special functions with ASAN?

Reply:
Updated with comment below:
Under ASAN generic memory usage functions provide information with ASAN overhead.